source/_patterns/00-protons/non-printing/_bootstrap-overrides.scss

```diff
@@ -0,0 +1,22 @@
+// Bootstrap variable customization
+// See all variables available at: ~bootstrap/scss/_variables.scss
+
+// Colors
+$theme-colors: (
+  'blerp': $color-blerp,
+  'derp': $color-derp,
+);
+
+// Fonts
+$headings-font-family: 'Oswald', sans-serif;
+
+// Buttons
+$btn-border-radius: 0.25rem;
+
+// Cards
+$card-border-width: 1px;
+
+// Bootstrap non-printing tools. DO NOT REMOVE.
+@import '~bootstrap/scss/functions';
+@import '~bootstrap/scss/variables';
+@import '~bootstrap/scss/mixins';
```

Comment:
@josemanosalvas, good catch. This broken out structure lets us do it the way you've mentioned. I'll update this stat.

Reply:
https://github.com/phase2/particle/issues/420

Reply:
Ah, yes, I remember now why it's in this order:

Bootstrap **really** wants variable overrides to come before Bootstrap imports for this reason:

![image](https://user-images.githubusercontent.com/295127/46748791-35243d80-cc69-11e8-802c-d6c93e28f512.png)

Since Bootstrap does processing on overridden vars the overrides have to come before. HOWEVER, this might be a workaround:

![image](https://user-images.githubusercontent.com/295127/46749149-00fd4c80-cc6a-11e8-877c-fa20882d007e.png)

This gives us the initial vars available to our overrides and then allows us to override. It's not **too** gross. What do you say @josemanosalvas ?

Reply:
Seems like a good solution to me!

Reply:
@josemanosalvas so, bad news ...

Doing this kind of double-`_variables.scss` import seems to cause a divergence in Bootstrap. receiving different values for `theme-color('primary');` if there exists this twice-imported `_variables.scss`.

Still investigating but leaning to only importing AFTER overrides (what is already in this PR).

Reply:
@illepic, @josemanosalvas, my solution so far has to been to just move the `_functions` and `_mixins` imports below the bootstrap override. Still doesn't quite fix `_colors`, `_spacing`, and `_type`, but there's really only so much you can do to pull these things out.